documentation/src/main/markdown/integrations/azureplugin/configuringandrunning.md

```diff
@@ -17,21 +17,21 @@ Configure your [detect_product_long] for Azure DevOps plugin by adding configura
 1. **Display name:** Type a unique name in this field.  Note that the name you type here displays in the left panel; the default name is **Run [detect_product_short] for your build**.
 1. Click **+ New** to add a new **[bd_product_short] Service Endpoint** and then configure the details.
 1. Click **+ New** to add a new **[bd_product_short] Proxy Service Endpoint** and then configure the details.
-1. **Detect Version**: Version of the [detect_product_short] binary to use. Synopsys recommends using the latest but you can specify a version override if desired.
-1. **Detect Run Mode:** Select the run mode. If you select Use Airgap Mode, a [detect_product_short] Air Gap Jar Directory Path field opens in which you must specify the [detect_product_short] Air Gap Jar Path.
-1. **Detect Arguments**: Here you can include additional [detect_product_short]* arguments; [detect_product_short] picks up your build environment variables and your project variables. Use a new line or space to separate multiple arguments. Use double quotes to escape. You can use environment and build variables.
+1. **[detect_product_short] Version**: Version of the [detect_product_short] binary to use. Synopsys recommends using the latest but you can specify a version override if desired.
```

Comment:
Remove Synopsys reference.

Reply:
How did I let that by before... yikes!
Thanks.

Reply:
Done.